src/EmployeesAdministrationApp.java

```diff
@@ -62,7 +64,7 @@ private static void loadFile() {
 					listOfEmployees.clear();
 					JSONArray arrayFromJsonFile = (JSONArray) parser.parse(new FileReader(jsonFile));
 					for (Object emp : arrayFromJsonFile) {
-						listOfEmployees.add(emp);
+						listOfEmployees.add((HashMap<String, Object>) emp);
```

Comment:
this should be changed

Reply:
I changed the list of employees from List to HashMap